scripts/build-combined-fwbundle.sh

```diff
@@ -62,8 +62,7 @@ for REV in $BOARD_REVS; do
   fi
 
   echo "Building $BOARD"
-  west build -d "${TEMP_DIR}${REV}" --sysbuild -p -b "$BOARD" app/smc \
-    >/dev/null 2>&1
+  west build -d "${TEMP_DIR}${REV}" --sysbuild -p -b "$BOARD" app/smc
```

Comment:
It may be better if standard error and standard output were redirected to a file, and in the case of an error, a trap / signal handler were to print the contents of that file.

The redirections were mainly intended to reduce verbosity, so I would like to preserve that if possible.